docs/transactions.md

```diff
@@ -0,0 +1,113 @@
+# Transactions v2
+
+## URL
+
+### Account transactions
+
+All Transactions from a given Account:
+`https://rest.opencollective.com/v2/{slug}/transactions.csv`
+
+E.g. https://rest.opencollective.com/v2/babel/transactions.csv
+
+### Host transactions
+
+All Transactions accounted by a given Fiscal Host:
+`https://rest.opencollective.com/v2/{slug}/hostTransactions.csv`
+
+E.g. https://rest.opencollective.com/v2/opensource/hostTransactions.csv
+
+### Parameters
+
+- `limit`: default is `1000`, maximum is `10000`
+- `offset`: default is 0
+- `type`: `CREDIT` or `DEBIT`
+- `kind`: comma separated list of `KIND`s
+- `dateFrom`: transactions after UTC date
+- `dateTo`: transactions before UTC date
+- `minAmount`: transactions more than the amount (warning, in cents!)
+- `maxAmount`: transactions less than the amount (warning, in cents!)
+- `includeIncognitoTransactions`: include incognito transactions made by the account (only for authenticated user)
+- `includeChildrenTransactions`: include transactions by children of the account (projects and events)
+- `includeGiftCardTransactions`: include transactions with Gift Cards issued by the account
+
+### Authentication
+
+Create an Api Key and pass it in the URL parameters as `apiKey`.
```

Comment:
Would be great to have a link, for someone external it would require some digging to know where they should go from here. I just looked and didn't find any docs page about it, but we could easily create one based on the content from https://medium.com/open-collective/open-collective-graphql-api-preview-3b42ed1d55ff.

Reply:
An update to our API docs is due! Adding a link to create an Api Key.